pkg/rbac/rbac.go

```diff
@@ -0,0 +1,62 @@
+package rbac
+
+import (
+	"github.com/iota-uz/iota-sdk/modules/core/domain/aggregates/user"
+	"github.com/iota-uz/iota-sdk/modules/core/domain/entities/permission"
+)
+
+type Permission interface {
```

Comment:
Since you already created this package, please move the RBAC manager from `core/domain/entities/permissions` here